src/tests/ftest/dfuse/read.py

```diff
@@ -0,0 +1,96 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+
+from dfuse_test_base import DfuseTestBase
+from run_utils import run_remote
+
+
+class DFuseReadTest(DfuseTestBase):
+    """Base ReadTest test class.
+    :avocado: recursive
+    """
+
+    def test_dfuse_pre_read(self):
+        """
+        Test Description:
+            Ensure that pre-read feature is working.
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=dfuse
+        :avocado: tags=DFuseReadTest,test_dfuse_pre_read
+        """
+
+        pool = self.get_pool(connect=False)
+        container = self.get_container(pool)
+
+        self.load_dfuse(self.hostlist_clients, None)
+
+        self.dfuse.disable_wb_cache.value = True
```

Comment:
If this is always on, it's better to set in the config as
```
dfuse:
  disable_wb_cache: true
```

Reply:
Done.